yard-api/src/main/java/org/kie/yard/api/model/DecisionLogic.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
```

Comment:
i guess those type of license header could be kept unless we are licensing the project under different license? WDYT?

Reply:
Maybe. The old Red Hat one we had was 
```
 * Copyright 2017 Red Hat, Inc. and/or its affiliates.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
```
I am wondering if there is a legal difference that might matter somehow.

Reply:
i guess that Copyright is not needed imho, the rest if the project is under Apache License is or better to check what the license.
I created completely new repository with suggested Apache licensed code here https://github.com/mareknovotny/test-license/blob/main/LICENSE